examples/coronavirustwittermap/web/conf/application-https.conf

```diff
@@ -0,0 +1,172 @@
+# This is the main configuration file for the application.
+# ~~~~~
+
+# Secret key
+# ~~~~~
+# The secret key is used to secure cryptographics functions.
+# If you deploy your application to several instances be sure to use the same key!
+play.crypto.secret = "Yf]0bsdO2ckhJd]^sQ^IPISElBrfy<XWdTWukRwJK8KKc3rFG>Cn;nnaX:N/=R1<"
+
+# The application languages
+# ~~~~~
+play.i18n.langs = ["en"]
+
+# Global object class
+# ~~~~~
+# Define the Global object class for this application.
+# Defaults to Global in the root package.
+# application.global=my.Global
+
+# Router
+# ~~~~~
+# Define the Router object to use for this application.
+# This router will be looked up first when the application is starting up,
+# so make sure this is the entry point.
+# Furthermore, it's assumed your route file is named properly.
+# So for an application router like `my.application.Router`,
+# you may need to define a router file `conf/my.application.routes`.
+# Default to Routes in the root package (and conf/routes)
+# application.router=my.application.Routes
+
+# Database configuration
+# ~~~~~
+# You can declare as many datasources as you want.
+# By convention, the default datasource is named `default`
+#
+# db.default.driver=org.h2.Driver
+# db.default.url="jdbc:h2:mem:play"
+# db.default.user=sa
+# db.default.password=""
+
+# Evolutions
+# ~~~~~
+# You can disable evolutions if needed
+# evolutionplugin=disabled
+
+# Uncomment this for the most verbose Akka debugging:
+akka {
+  loglevel = "DEBUG"
+  actor {
+    debug {
+      receive = on
+      autoreceive = on
+      lifecycle = on
+    }
+    mailbox {
+      requirements {
+        "akka.dispatch.BoundedDequeBasedMessageQueueSemantics" = akka.actor.mailbox.bounded-deque-based
+      }
+    }
+  }
+}
+
+play {
+  ws {
+    timeout {
+      connection = 60000
+      idle = 600000
+    }
+  }
+}
+
+bounded-mailbox {
+  mailbox-type = "akka.dispatch.BoundedDequeBasedMailbox"
+  mailbox-capacity = 2000
+  mailbox-push-timeout-time = 10s
+}
+
+# Web page title of the application
+app.name = "CoronaVirusTwittermap"
+
+cloudberry.register = "https://localhost:9000/admin/register"
```

Comment:
If the only changes are these 4 lines, will it be better to put them in the existing application.conf so that it's easier to maintain?